docs/content/development/experimental.md

```diff
@@ -12,3 +12,31 @@ druid.extensions.coordinates=["io.druid.extensions:druid-histogram:{VERSION}"]
 ```
 
 The configuration for all the indexer and query nodes need to be updated with this.
+
+
+#### KafkaSimpleConsumerFirehose
```

Comment:
Are other experimental features' doc under "druid/docs/content/development/"? If not, could you point me to an example that I could follow? Thanks